### PR TITLE
Show message timestamp in action menu

### LINKED ISSUE #8267: Show message timestamp in action menu
### Is your feature request related to a problem? Please describe.

If I look at my inbox I see a relative time indicator for when a message was sent, but not always in a precise format. sometimes it is the time, sometimes just the day. As a user I want to see at which time a message was sent, regardless if it's recent old older.

Hovering the time doesn't give more info because then the time is replaced with the menu handle

Bildschirmfoto vom 2023-03-22 17-21-30

### Describe the solution you'd like

Include the timestamp in the actions menu like Talk does

Bildschirmfoto vom 2023-03-22 17-18-32

ref https://github.com/nextcloud/spreed/pull/9003

### Describe alternatives you've considered

_No response_

### Additional context

@jancborchardt @nimishavijay yay or nay?

### PR DESCRIPTION
**Pull request by hamza221**
closes #8267
image

### CODE REVIEW

File: src/components/Envelope.vue
Comment (ChristophWurst) [inline src/components/Envelope.vue:527]: make this a computed prop

**Review by ChristophWurst — COMMENTED** [review]
Awesome! Let's wait for the approval of the designers

**Review by nimishavijay — CHANGES_REQUESTED** [review]
Looks good! One suggestion: add a [clock icon](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aschedule%3A) on the left of the item so that all the text is nicely aligned :)

File: src/util/shortRelativeDatetime.js
Comment (ChristophWurst) [inline src/util/shortRelativeDatetime.js:46]: this is a tricky topic. I think the format should depend on the user's locale. Let's go with the format `lll`. See https://momentjs.com/ for examples.

**Review by nimishavijay — APPROVED** [review]
Nice! 🚀

**Pull request merged**
